src/network/message.rs

```diff
@@ -188,12 +198,16 @@ impl NetworkMessage {
             NetworkMessage::Alert(_)    => "alert",
             NetworkMessage::Reject(_)    => "reject",
             NetworkMessage::FeeFilter(_) => "feefilter",
+            NetworkMessage::Unknown { .. } => "unknown",
```

Comment:
shouldn't this return the internal command and not `unknown` string?

Reply:
Well cmd is the one that returns a static string, which is what we use to create CommandString for outgoing messages. Since CommandString is a `Cow<'static, str>`, using the static reference prevents an allocation of the command. So I think `cmd` staying static is useful. If really need be, we could make it a private method and panic when it's called with "unknown" because internally we never call it for unknown messages.

Reply:
Anther panic does not sound good